GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockVirtualizationInstance.cs

```diff
@@ -174,12 +149,61 @@ public void WaitForCreateWriteBuffer()
             this.waitForCreateWriteBuffer.WaitOne();
         }
 
+        public HResult CompleteCommand(int commandId, NotificationType newNotificationMask)
+        {
+            this.commandCompleted.Set();
+            return HResult.Ok;
+        }
+
+        public HResult CompleteCommand(int commandId, IDirectoryEnumerationResults results)
+        {
+            this.commandCompleted.Set();
+            return HResult.Ok;
+        }
+
+        HResult IVirtualizationInstance.CompleteCommand(int commandId, HResult completionResult)

```

Comment:
Why did you need to include the `IVirtualizationInstance.` here and below?

Reply:
Because I was being silly!